### PR TITLE
Add back certmanager tests

### PR DESCRIPTION
**Pull request by skonto**
**Which issue(s) this PR fixes**:<br>
Fixes https://github.com/knative/serving/issues/15236

### CODE REVIEW

File: prow/jobs_config/knative/serving.yaml [hunk 1]
```diff
@@ -183,13 +183,13 @@ jobs:
     requirements: [perf]
     command: [runner.sh, ./test/presubmit-tests.sh, --run-test, ./test/performance/performance-tests.sh]
 
-  # - name: certmanager-integration-tests
-  #   types: [presubmit]
-  #   command: [runner.sh, ./test/e2e/certmanager/presubmit-tests.sh, --integration-tests]
+  - name: certmanager-integration-tests
+    types: [presubmit]
+    command: [runner.sh, ./test/e2e/certmanager/presubmit-tests.sh, --run-test, ./test/e2e/certmanager/e2e-tests.sh]
```
Comment (skonto) [inline prow/jobs_config/knative/serving.yaml:188]: Let's see how that goes. Gradually we should move the script to the top dir and remove any extra sciprts.

Comment (dprotaso) [reply prow/jobs_config/knative/serving.yaml:188]: Can you make an issue for that?

**Pull request merged**